vividus-plugin-sauce-labs/build.gradle

```diff
@@ -0,0 +1,14 @@
+project.description = 'Vividus plugin for Sauce Labs'
+
+dependencies {
+    implementation project(':vividus-plugin-web-app')
+    implementation project(':vividus-bdd-engine')
+    implementation project(':vividus-extension-selenium')
+
+    implementation(group: 'javax.inject', name: 'javax.inject', version: versions.javaxInject)
```

Comment:
could you please use constructor based injections

Reply:
done